frontend/chat-plugin/src/airyRenderProps/AiryHeaderBar/index.tsx

```diff
@@ -0,0 +1,29 @@
+import {h} from 'preact';
+import style from './index.module.scss';
+
+type Props = {
+  toggleHideChat: () => void;
```

Comment:
@AitorAlgorta I removed the org image logic here, because we do not have it in the API